declarative-pipeline-migration-assistant-api/src/main/java/io/jenkins/plugins/todeclarative/converter/api/ConverterResult.java

```diff
@@ -18,6 +18,8 @@
 
     private List<Supplier<ModelASTTreeStep>> wrappingTreeSteps = new ArrayList<>();
 
+    private List<String> convertedPlugins = new ArrayList<>();
```

Comment:
Since the values are class names of specific `BuildStep`s, `BuildWrapper`s, `Publisher`s, etc., I think it is really confusing to have `plugins` in the name of this variable and the related methods. Maybe call it `convertedTypes` or something instead? Thoughts?

Reply:
moved to #20